src/main/cpp/LED.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2023-2024 CurtinFRC
+// Open Source Software, you can modify it according to the terms
+// of the MIT License at the root of this project
+
+#include "LED.h"
+
+LED::LED() {}
+
+void LED::OnUpdate(units::second_t dt) {
+  switch (_state) {
+    case LEDState::kIdle: {
+      _led.frc::PWM::SetSpeed(0);
```

Comment:
What is this doing? you appear to be getting a field then modifying a static member on that field...